package.json

```diff
@@ -65,6 +65,8 @@
     "@typescript-eslint/eslint-plugin": "^5.42.1",
     "@typescript-eslint/parser": "^6.19.0",
     "@vitejs/plugin-react": "^4.0.0",
+    "arconnect": "^1.0.3",
```

Comment:
Shouldn't this be a regular dependency vs. dev dependency?

Reply:
the name is misleading, its a types package